HACKING.md

```diff
@@ -1,94 +1,48 @@
-* An admin-scoped `KUBECONFIG` for the cluster.
-* The [operator-sdk](https://github.com/operator-framework/operator-sdk).
-
-#### GCP test clusters
-
-One reliable and hands-free way to create a suitable test cluster and `KUBECONFIG` in GCP is to use the [openshift/release](https://github.com/openshift/release/tree/master/cluster/test-deploy) tooling. The default `gcp-dev` profile will produce a cluster compatible with Service Load Balancers.
-
-### Building
-
-To build the operator during development, use the standard Go toolchain:
+To build the operator, run:
 
 ```
-$ go build ./...
+$ make build
 ```
 
-### Container image
-
-To build the operator docker image:
+## Developing
 
-##### Using docker
-
-```
-docker build -t openshift/cluster-ingress-operator:latest .
-```
+### Prerequisites
 
-##### Using buildah
+* An [OpenShift cluster](https://github.com/openshift/installer)
+* An admin-scoped `KUBECONFIG` for the cluster.
 
-```
-sudo buildah bud -t openshift/cluster-ingress-operator:latest .
-```
+Uninstall the existing operator and all its managed components:
 
 ```
-sudo buildah push openshift/cluster-ingress-operator:latest docker-daemon:openshift/cluster-ingress-operator:latest
+$ hack/uninstall.sh
 ```
 
-
-### Running
-
-To run the operator, first deploy the custom resource definitions:
+Build a new image and custom manifests:
 
 ```
-$ oc create -f manifests/02_crd.yaml
+$ REPO=docker.io/you/origin-cluster-ingress-operator make release-local
 ```
 
-#### Running with Operator SDK
-
-Use the operator-sdk to launch the operator:
+Follow the instructions to install the operator, e.g.:
 
 ```
-$ operator-sdk up local namespace default --kubeconfig=$KUBECONFIG
+$ oc apply -f /tmp/manifests/path
 ```
 
-If you're using the `openshift/release` tooling, `KUBECONFIG` will be something like `$RELEASE_REPO/cluster/test-deploy/gcp-dev/admin.kubeconfig`.
+## Tests
 
-#### Running a Containerized Operator
+Run unit tests:
 
 ```
-oc create -f manifests/04_cluster-ingress-operator.yaml
+$ make test
 ```
 
-*In order to run as a container you will need to upload the Cluster Ingress Operator image to your registry*
-
-
-## Tests
-
-### Integration tests
-
-Integration tests are still very immature. To run them, start with an OpenShift 4.0 cluster and then run the following,
-substituting for your own details where appropriate. This assumes `KUBECONFIG` is set.
+Assuming `KUBECONFIG` is set, run end-to-end tests:
 
 ```
-# 1. Uninstall any existing cluster-version-operator and cluster-ingress-operator.
-$ hack/uninstall.sh
-
-# 2. Build and push a new cluster-ingress-operator image.
-$ REPO=docker.io/username/origin-cluster-ingress-operator make release-local
-
-# 3. Run `oc apply` as instructed to install the locally-built operator.
-
-# 4. Run integration tests against the cluster.
-$ CLUSTER_NAME=your-cluster-name make test-integration
+$ make test-e2e
```

Comment:
Should we say anything about running the origin tests?

Reply:
Maybe -- I'd like to think more about it and do some followup improvements here and in the DNS operator doc with other test info and also some alternative development workflows